api_helpers.py

```diff
@@ -284,12 +286,15 @@ def create_hs_res_from_czo_row(czo_res_dict, czo_hs_account_obj, index=-99, ):
         # parse czo_id
         czo_id = _extract_value_from_df_row_dict(czo_res_dict, "czo_id")
         record_dict["czo_id"] = czo_id
-        logging.info("Working on Row {index} CZO_ID {czo_id}".format(index=index, czo_id=czo_id))
+        logging.info("Working on NO.{index} CZO_ID {czo_id}".format(index=index, czo_id=czo_id))
 
         # parse CZOS
         czos = _extract_value_from_df_row_dict(czo_res_dict, "CZOS")
         czos_list = czos.split('|')
         czo_primary = czos_list[0]
+        if len(czos_list) > 1:
+            czo_primary = "national"  # cross-czo res goes to national account
+            logging.info("Cross-CZO resource to be created by National account: {}".format(czos_list))
 
```

Comment:
We do a ton of logging, can we change our attitude to assume success, but have everything with try... except so we assume success and log/catch errors?

Reply:
I personally prefer to keep these loggings. Since they are verbose, maybe we should assign them with "Debug" level and output to a separate file "xxxx_verbose.log"